test/storage/sequelize.test.ts

```diff
@@ -8,6 +8,12 @@ import { join } from 'path';
 import { v4 as uuid } from 'uuid';
 import jetpack = require('fs-jetpack');
 
+// TODO [sequelize@>=6.0.0] remove when sequelize fixes the types bug
+// a change in the types of sequelize@next makes `describe` a static function - see https://github.com/sequelize/sequelize/issues/12296
+// eslint-disable-next-line @typescript-eslint/ban-ts-ignore
+// @ts-ignore
+const describeModel = (model: sequelize.Model) => model.describe();
```

Comment:
```suggestion
const describeModel = (model: typeof sequelize.Model) => model.describe();
```

Reply:
This doesn't work because we don't have a `typeof Model`, we have a `Model` - this is called with the return value of `model.sync()` - from sequelize: https://github.com/sequelize/sequelize/blob/b2bccb8aa3faeb6a891fd9f7d3858ebff5b9c32f/types/lib/model.d.ts#L1604-L1608

I'm going to change to `any` for now. It might be that we are using `describe` wrong and it's working by chance, but this behaviour is currently going to make every PR fail, so I've changed the todo to `TODO [>=3.0.0]`, i.e. it's on us to fix before we ship v3.

Reply:
Hmm, great catch, sorry about that, I am pretty sure the `sync` type signature is the one wrong then. I will look into it later.